lib/cargo.js

```diff
@@ -108,6 +108,21 @@ export function provideBuilder() {
           args: [ 'update' ].concat(args),
           sh: false,
           errorMatch: []
+        },
+        {
+          name: `Cargo: build example`,
+          exec: cargoPath,
+          args: [ 'build', '--example', '{FILE_ACTIVE_NAME_BASE}' ].concat(args),
```

Comment:
I just realized [these replacement rules](https://github.com/noseglid/atom-build#replacements) I was not aware of.